src/stk/molecular/topology_graphs/polymer/linear/linear.py

```diff
@@ -4,10 +4,13 @@
 
 """
 
+from __future__ import annotations
+
 from dataclasses import dataclass
+from typing import Iterable, Tuple
```

Comment:
do not use `typing.Tuple` using `tuple`  (that's why the `__future__` import is there)

Reply:
That was you ;)

Reply:
to be fair, that was before i was doing the future imports